processors/filterprocessor/fields/searcher.go

```diff
@@ -1,12 +1,12 @@
 package fields
 
+//go:generate go run ../gen/generate.go fields ./generated_signed_txn_map.go
+
 import (
 	"fmt"
-	"reflect"
-	"strings"
-
 	"github.com/algorand/go-algorand/data/transactions"
 	"github.com/algorand/indexer/processors/filterprocessor/expression"
+	"reflect"
```

Comment:
nit: I think the `reflect` import should go back where it came from - right after the `fmt` import

Reply:
Done